build/ci/tools/osuosl/index.html

```diff
@@ -1,52 +1,71 @@
-<p>A development build can run side-by-side with other versions of MuseScore on the same computer. If you change preferences in the nightly build, it does not affect your preferences in the prerelease or stable versions of MuseScore. The development builds use a cymbal icon (a play on the fact that nightly builds are likely to suspend and crash) to visually differentiate them from normal versions of MuseScore. If this sounds scary to you, instead use the latest stable version of MuseScore from <a href="https://www.musescore.org/">musescore.org</a>.</p>
-<h2>Windows</h2>
-<h3>3.x</h3>
-<ul>
-<li><a href="windows/3x/nightly/?C=M;O=D">nightly</a></li>
-<li><a href="windows/3x/testing/?C=M;O=D">beta</a></li>
-<li><a href="windows/3x/stable/?C=M;O=D">stable</a></li>
-</ul>
-<h3>4.x (master)</h3>
-<ul>
-<li><a href="windows/4x/nightly/?C=M;O=D">nightly</a></li>
-<li><a href="windows/4x/testing/?C=M;O=D">beta</a></li>
-<li><a href="windows/4x/stable/?C=M;O=D">stable</a></li>
-</ul>
-<h2>Linux</h2>
-<h3>3.x</h3>
-<ul>
-<li><a href="linux/3x/nightly/?C=M;O=D">nightly</a></li>
-<li><a href="linux/3x/testing/?C=M;O=D">beta</a></li>
-<li><a href="linux/3x/stable/?C=M;O=D">stable</a></li>
-</ul>
-<h3>4.x (master)</h3>
-<ul>
-<li><a href="linux/4x/nightly/?C=M;O=D">nightly</a></li>
-<li><a href="linux/4x/testing/?C=M;O=D">beta</a></li>
-<li><a href="linux/4x/stable/?C=M;O=D">stable</a></li>
-</ul>
-<h2>MacOS</h2>
-<h3>3.x</h3>
-<ul>
-<li><a href="macos/3x/nightly/?C=M;O=D">nightly</a></li>
-<li><a href="macos/3x/testing/?C=M;O=D">beta</a></li>
-<li><a href="macos/3x/stable/?C=M;O=D">stable</a></li>
-</ul>
-<h3>4.x (master)</h3>
-<ul>
-<li><a href="macos/4x/nightly/?C=M;O=D">nightly</a></li>
-<li><a href="macos/4x/testing/?C=M;O=D">beta</a></li>
-<li><a href="macos/4x/stable/?C=M;O=D">stable</a></li>
-</ul>
+<p class="lead">
+The development builds allow you to test the very latest developments, including new features and bug fixes. 
+The goal is to catch new bugs, crashes, and regressions before they reach the prerelease or stable versions. 
+For more information, see <a href="https://musescore.org/en/developers-handbook/comparison-stable-beta-and-development-versions">Comparison of stable, beta, and nightly versions</a>. 
+You can report any regressions or bugs you find via <a href="https://musescore.org/en/project/issues/musescore" target="_blank">Issues Tracker</a>. 
+</p>
+<p class="lead">
+A development build can run side-by-side with other versions of MuseScore on the same computer. 
+If you change preferences in the nightly build, it does not affect your preferences in the prerelease or stable versions of MuseScore. 
+If this sounds scary to you, instead use the latest stable version of MuseScore from <a href="https://www.musescore.org/">musescore.org</a>.
```

Comment:
I don't think "If this sounds scary to you" should be kept either if the previous sentence is removed?

Reply:
I agree, I will remove it